lib/MenuItem.php

```diff
@@ -14,7 +14,7 @@
  */
 class MenuItem extends CoreEntity {
 
-	protected WP_Post $core_object;
+	protected WP_Post $wp_object;
```

Comment:
Update with proposed name (@gchtr in #2560)

Reply:
Thanks!